geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/ExecuteFunction65.java

```diff
@@ -52,7 +56,24 @@ public static Command getCommand() {
     return singleton;
   }
 
-  ExecuteFunction65() {}
+  private final InternalFunctionExecutionService internalFunctionExecutionService;
+  private final ServerToClientFunctionResultSender65Factory serverToClientFunctionResultSender65Factory;
+  private final FunctionContextImplFactory functionContextImplFactory;
+
+  private ExecuteFunction65() {
+    this(InternalFunctionService.getInternalFunctionExecutionService(),
+        new DefaultServerToClientFunctionResultSender65Factory(),
+        new DefaultFunctionContextImplFactory());
+  }
+
+  @TestingOnly
```

Comment:
Same comment as above.  It would be nice to avoid a test only constructor if possible.